docs/src/man/hosting.md

```diff
@@ -441,6 +441,17 @@ The `permissions:` line above is described in the
 an alternative is to give GitHub workflows write permissions under the repo settings, e.g.,
 `https://github.com/<USER>/<REPO>.jl/settings/actions`.
 
+## JuliaHub
+
+[JuliaHub](https://juliahub.com/ui/Home) provides an [overview on registered Julia packages](https://https://juliahub.com/ui/Packages).
```

Comment:
"an overview *of*", not "*on*", right?

Reply:
"of" https://www.merriam-webster.com/dictionary/overview#examples